This patch adds libpostproc to the list of ffmpeg's module checked and linked
against by VLC's configure script.  This is necessary as newer versions of
ffmpeg does not link libavutil/libavcodec against libpostproc, so the postproc
symbols will be missing and VLC won't be able to load ffmpeg's plugin.

See Gentoo Bug #125071.

Index: vlc-0.8.4a/configure.ac
===================================================================
--- vlc-0.8.4a.orig/configure.ac
+++ vlc-0.8.4a/configure.ac
@@ -2206,7 +2206,7 @@ dnl Those options have to be here becaus
 	fi])
 
 dnl Trying with pkg-config
-   PKG_CHECK_MODULES(FFMPEG,[libavcodec, libavformat],
+   PKG_CHECK_MODULES(FFMPEG,[libavcodec, libavformat, libpostproc],
     [
      AC_CHECK_HEADERS(ffmpeg/avcodec.h)
      AC_CHECK_HEADERS(postproc/postprocess.h)
